quodlibet/cli.py

```diff
@@ -239,6 +241,13 @@ def queue(*args):
             queue(command, filename)
         elif command == "enqueue-files":
             queue(command, arg)
+        elif command == "set-pattern":
+            try:
+                queue(command, args[0])
+            except IndexError:
+                queue(command)
+        elif command == "set-pattern-help":
```

Comment:
This isn't the approach taken by any of the other CLI commands. I would try to remove this and fit a short, useful help message in the `help` parameter for `set-pattern` itself. The more expansive explanation for how to use the option should go in the documentation.